src/utils/react.ts

```diff
@@ -46,13 +46,17 @@ export function fakeRenderComponent(fun: Function, customHooks: any = {}): any {
 }
 
 export function wrapReactType(node: any, prop: any = 'type') {
-  return (node[prop] = { ...node[prop] });
+  return node[prop].__DECKY_WRAPPED ? node[prop] : (node[prop] = { ...node[prop], __DECKY_WRAPPED: true });
 }
 
 export function wrapReactClass(node: any, prop: any = 'type') {
-  const cls = node[prop];
-  const wrappedCls = class extends cls {};
-  return (node[prop] = wrappedCls);
+  if (node[prop].__DECKY_WRAPPED) {
+    return node[prop];
+  } else {
+    const cls = node[prop];
+    const wrappedCls = class extends cls { __DECKY_WRAPPED = true; };
```

Comment:
shouldn't __DECKY_WRAPPED be static to make this work (or is it already?)

Reply:
shoot, yea it should. will change in a sec